vortex-array/src/array/chunked/compute/min_max.rs

```diff
@@ -0,0 +1,57 @@
+use vortex_error::{vortex_err, VortexResult};
+use vortex_scalar::Scalar;
+
+use crate::array::{ChunkedArray, ChunkedEncoding};
+use crate::compute::{min_max, MinMaxFn, MinMaxResult};
+use crate::{partial_max, partial_min};
+
+impl MinMaxFn<ChunkedArray> for ChunkedEncoding {
+    fn min_max(&self, array: &ChunkedArray) -> VortexResult<Option<MinMaxResult>> {
+        let res = array
+            .array_iterator()
+            .map(|chunk| {
+                let chunk = chunk?;
+                if let Some(min_max) = min_max(chunk)? {
+                    Ok((Some(min_max.min), Some(min_max.max)))
+                } else {
+                    Ok((None, None))
+                }
+            })
+            .collect::<VortexResult<Vec<_>>>()?;
```

Comment:
Is there no try_fold or whatever to do this all in one?

Reply:
If we allow unstable